tests/integration-v1/cattletest/core/test_private_repos.py

```diff
@@ -49,7 +49,7 @@ def test_create_container_with_registry_credential(client, context):
     assert container is not None
     assert container.registryCredentialId == reg_cred.id
     assert container.startOnCreate is False
-    assert container.imageUuid == uuid+":latest"
```

Comment:
we no longer add tag for image uuid. If it doesn't have a tag, leave it as it is.